zerver/tests/test_notifications.py

```diff
@@ -323,3 +328,24 @@ def test_deleted_message_in_huddle_missed_stream_messages_as_user(self):
     def test_deleted_message_in_huddle_missed_stream_messages(self):
         # type: () -> None
         self._deleted_message_in_huddle_missed_stream_messages(False)
+
+    @patch('zerver.lib.email_mirror.generate_random_token')
+    def test_realm_emoji_in_missed_message(self, mock_random_token):
+        # type: (MagicMock) -> None
+        # Create a test emoji.
+        email = self.example_email('iago')
+        self.login(email)
+        with get_test_image_file('img.png') as fp1:
+            emoji_data = {'f1': fp1}
+            self.client_put_multipart('/json/realm/emoji/test_emoji', info=emoji_data)
+        self.logout()
```

Comment:
This part is just to ensure there exists at least one realm emoji?

It's possible from a test infrastructure perspective, we should add at least one realm emoji for the `zulip` realm to `populate_db`, so we don't have to do this setup work in every test that wants a realm emoji.  

Could be a good follow-up project.